tests/unittests/BackendTest.cpp

```diff
@@ -109,15 +109,15 @@ TEST_P(BackendTest, simpleInference) {
 
   auto *ex = mod.createVariable(ElemKind::IndexTy, {1, 1}, "exp");
 
-  auto *CV0 = F->createConv("conv1", input, 16, 5, 1, 2, 1);
+  auto *CV0 = F->createConv("conv1", input, 16, 5, 1, {2, 2, 2, 2}, 1);
```

Comment:
nit: you can now leave tests unchanged, because we have `createConv`, which accepts single pad.